dev/tasks/conda-recipes/azure.linux.yml

```diff
@@ -52,3 +54,8 @@ jobs:
     env:
       CROSSBOW_GITHUB_TOKEN: $(CROSSBOW_GITHUB_TOKEN)
     displayName: Upload packages as a GitHub release
+
+  # Upload to custom anaconda channel
+  - script: |
+      anaconda -t $(CROSSBOW_ANACONDA_TOKEN) upload -l nightly --force build_artifacts/linux-64/*.tar.bz2
```

Comment:
https://dev.azure.com/ursa-labs/crossbow/_build/results?buildId=8372&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=3367cfe7-6aa5-5a88-c888-e6814d74f6eb&l=16

I don't know why but `tar.bz2` can't be found...